server/src/index.js

```diff
@@ -11,10 +11,12 @@ const mocks = {
     author: () => {
       return {
         name: 'Grumpy Cat',
-        photo: 'https://res.cloudinary.com/dety84pbu/image/upload/v1606816219/kitty-veyron-sm_mctf3c.jpg',
+        photo:
```

Comment:
dont know why prettier needed to include this file 😠

Reply:
seems to want it in a new line 🤔